src/views/Dissolution/DissolutionReviewConfirm.vue

```diff
@@ -375,6 +375,16 @@ export default class DissolutionReviewConfirm extends Mixins(DateMixin) {
     })
   }
 
+  /** Called when component is mounted. */
+  mounted (): void {
+    this.setCertifyState(
+      {
+        valid: this.getCertifyState.valid,
+        certifiedBy: this.getCertifyState.certifiedBy
+      }
+    )
+  }
```

Comment:
Theoretically, this code does nothing -- the certify state was already set in fetchData().

This code _might_ hide the timing issue, but it's not a reliable solution.

I wonder if it's a reactive problem: the Certify component doesn't realize this object has been updated. But theoretically, all store objects are reactive.

Hmmm...

Reply:
For sure, can look at it further 👍

Reply:
Removed this, as well as a reassignment in the parsing of a draft Dissolution.